js/views/contentScreen.js

```diff
@@ -45,12 +46,12 @@ var ContentScreen = React.createClass({
 
     return (
       <div className={this.props.screenClassName}>
+        <Watermark {...this.props} controlBarVisible={false} nonClickable={true}/>
```

Comment:
Agree on the empty clickUrl, I lost that check along the way.

About the content screen: since the watermark can be placed in different parts of the screen, if we make it fully functional on CC or share screens, it might interfere with other elements (both visually and functionally). Currently it looks like it is behind the content screen, blurred with the video.